aegis-web/yo/app/styles/form.css

```diff
@@ -232,3 +232,7 @@ textarea.form-control {
 .dark::placeholder {
   color: #fff !important;
 }
+
+select {
+  text-transform: uppercase;
+}
```

Comment:
This applies to all `<select>` elements in aegis. Are you sure you want this?

Reply:
removed